visualc/VS2010/aescrypt2.vcxproj

```diff
@@ -91,10 +91,12 @@
     <ClCompile>
       <PrecompiledHeader>
       </PrecompiledHeader>
-      <WarningLevel>Level3</WarningLevel>
+      <WarningLevel>Level4</WarningLevel>
       <Optimization>Disabled</Optimization>
       <PreprocessorDefinitions>WIN32;_DEBUG;_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <AdditionalIncludeDirectories>../../include</AdditionalIncludeDirectories>
+      <TreatWarningAsError>true</TreatWarningAsError>
+      <DisableSpecificWarnings>4706;4127</DisableSpecificWarnings>

```

Comment:
[4706](https://msdn.microsoft.com/en-us/library/7hw7c1he.aspx?f=255&MSPPError=-2147217396) - assignment within conditional expression. Disabled, because that approach seems to be used widely across mbedTLS and it didn't seem worth it to change. Please let me know, if I'm wrong.
[4127](https://msdn.microsoft.com/en-us/library/6t66728h.aspx) - conditional expression is constant. Appears in some macros, so cannot be entirely cleaned up, plus the warning seems harmless.